src/tech/data-standards.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Data standards at dxw
+permalink: /:path/:basename/
+---
+In keeping with the GOV.UK [Service Manual](https://www.gov.uk/service-manual/service-standard/point-13-use-common-standards-components-patterns), dxw adheres to relevant open data standards wherever possible. While such standards are often recommended or mandated by government requirements, they are frequently useful in private and third-sector work as well.
+
+## Why standards are important
+
+There are three chief reasons why using data standards makes sense.
+
+First, organisations - and in particular, government - are becoming ever more aware of the importance of data. The advent of machine learning and artificial intelligence technologies, alongside the steadily growing importance of statistics in evidence-driven decision- and policy-making, means that not only survey data, but also routine administrative data, need to be treated as a potentially valuable resource. Care needs to be taken to ensure that data is collected in ways that ensure it is consistent, coherent, and comparable across the whole of UK public services.
+
+Second, the use of data standards supplies us with predefined resources that help us ensure high-quality data collection without the need for expensive, time-consuming, and error-prone research on our part. There is no point in developing, for example, our own set of bands for classifying users by age when [publicly-available standardised sets](https://analysisfunction.civilservice.gov.uk/policy-store/age-and-date-of-birth/) have already been defined for a variety of use-cases. This is particularly the case when collecting data concerning protected characteristics (for example, religion, ethnicity, or sexuality) where understandings evolve over time, categories are sometimes contested, and personal sensitivities are likely to be high. 
+
+Finally, data standards can help systems interoperate, reducing the risk of silos and the need for middleware to bridge gaps in information exchange.
+
+## Which standard to use
+
+Choosing and/or finding a relevant data standard can be difficult. As the old joke goes, _standards are great – that's why there are so many of them_. This is particularly the case in government, where particular ministries or councils will often develop and promote their own bespoke ways to capture data _ad hoc_ or to suit their own very particular needs.
+
+### Mandated standards
+
+For some well-defined areas, the UK government mandates the use of [particular standards](https://www.gov.uk/government/publications/open-standards-for-government). These are all widely used and well-tested and are suitable for use on almost any project, whether for governmental or other organisations.
```

Comment:
'government' in 'UK government' should be capitalised to be consistent with the next section